frontend/src/AppRoutes/index.tsx

```diff
@@ -14,7 +14,12 @@ const App = (): JSX.Element => (
 			<Suspense fallback={<Spinner size="large" tip="Loading..." />}>
 				<Switch>
 					{routes.map(({ path, component, exact }, index) => (
-						<Route key={index} exact={exact} path={path} component={component} />
+						<Route
```

Comment:
can we use `<Route key={`${path}`} exact={exact} path={path} component={component} />` ?

Reply:
Yeah I thought of doing this. I ll change this quickly.

Reply:
I changed it to
```typescript
<Route key={`${path}`} ... />
```
instead of

```typescript
<Route key={path} ... />
```
because path variable is of type `string` and to use it as a key for the component, we need to change its type to `String`. We can use something like 

```typescript
<Route key={(String)(path)} ... />
```

but using template Strings seems better.

Reply:
Yeah `<Route key={`${path}`} ... />` will work